x-pack/plugins/apm/public/components/app/mobile/transaction_overview/index.tsx

```diff
@@ -63,6 +69,29 @@ export function MobileTransactionOverview() {
       <EuiFlexItem>
         <EuiHorizontalRule />
       </EuiFlexItem>
+      <EuiFlexItem>
+        <EuiFlexGroup>
+          <EuiFlexItem grow={8}>
+            <EuiPanel hasBorder={true}>
+              <LatencyMap
```

Comment:
fyi, the map was renamed in this [PR](https://github.com/elastic/kibana/pull/150040/files) and merged couples of minutes ago